server/routes/reporting/standard-report/standard-report-print.js

```diff
@@ -49,6 +49,10 @@ async function standardReportPrint(app, req, res, reportKey, data) {
           text = text === 'Civ' ? 'Civil' : 'Criminal';
         }
 
+        if (header.id === 'comments') {
+          text = text.replace('\n','\n\n')
```

Comment:
should this not be `replaceAll()` or use global regex `/\n/g`? could be that we need to replace more than one instance of `\n` for multiple groups of comments?

Reply:
you're right. updated.